Library/test-helpers/Dockerfile.webhook

```diff
@@ -1,12 +1,12 @@
 FROM registry.access.redhat.com/ubi9/ubi-minimal
 
 RUN microdnf makecache && \
-    microdnf install -y nmap-ncat && \
+    microdnf install -y openssl && \
     microdnf clean all && \
     rm -rf /var/cache/dnf/*
 
 ENV WEBHOOK_SERVER_PORT 8080
 
 EXPOSE $WEBHOOK_SERVER_PORT
 
-CMD ["/usr/bin/bash", "-c", "/usr/bin/ncat --no-shutdown -k -l -c '/usr/bin/sleep 3 && echo HTTP/1.1 200 OK' -o /var/tmp/webhook/revocation_log $WEBHOOK_SERVER_PORT"]
+CMD ["/usr/bin/bash", "-c", "openssl s_server -cert /var/tmp/webhook/server-cert.crt -key /var/tmp/webhook/server-private.pem -port ${WEBHOOK_SERVER_PORT} &> /var/tmp/webhook/revocation_log"]
```

Comment:
I propose to run it in debug mode.

Reply:
I'll add the option to run in debug mode